### PR TITLE
Main Activity Refactoring

### PR DESCRIPTION
**Pull request by jacobrs**
Simplified code for main activity and added automatic removal of extra 0 decimal. Still waiting on calculator implementation to test more.

### CODE REVIEW

File: app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt [hunk 1]
```diff
@@ -7,10 +7,10 @@ import java.util.*
...
 class MainActivity : AppCompatActivity() {
 
-    val calc = Calculator()
-    val numstack: Stack<Double> = Stack()
-    var lastOperator: Char? = null
-    var last: Double? = null
+    private val calc = Calculator()
+    private val digitStack: Stack<Double> = Stack()
+    private var lastOperator: Char? = null
+    private var last: Double? = null
```
Comment (fvalela1) [inline app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt:13]: non descriptive variable. Please meet standards of company.

File: app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt [hunk 2]
```diff
@@ -19,88 +19,108 @@ class MainActivity : AppCompatActivity() {
...
+        btn_8.setOnClickListener{ genericDigitListener(8.0) }
+        btn_9.setOnClickListener{ genericDigitListener(9.0) }
     }
 
-    fun refreshResult(value: Double) {
-        result.text = value.toString()
+    private fun setOperatorListeners() {
+        btn_plus.setOnClickListener{ pushOperator('+'); }
+        btn_minus.setOnClickListener{ pushOperator('-'); }
+        btn_root.setOnClickListener{ pushOperator('√'); }
```
Comment (fvalela1) [inline app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt:120]: Are we or are we not removing the square root

**Review by fvalela1 — APPROVED** [review]

File: app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt (hunk 2 above)
Comment (jacobrs) [reply app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt:120]: yeah, we can replace it by undo.

File: app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt (hunk 1 above)
Comment (jacobrs) [reply app/src/main/java/com/example/francescovalela/androidtutorialwinter2018/MainActivity.kt:13]: SCS Coding Standards too harsh... but you're right pretty non descriptive

**Pull request merged**